bids_prov/tests/test_spm_parser.py

```diff
@@ -90,9 +90,7 @@ def test_get_input_entity():
     entities = [{
         "@id": "urn:c15521b1-b3dc-450a-9daa-37e51b591d75",
         "label": "func_sub-01_task-tonecounting_bold_trunctest.nii.gzs",
-        "prov:atLocation": "ds011/sub-01/func/sub-01_task-tonecounting_bold_trunctest.nii.gzs",
-        'digest': {
-            'sha256_urn:c15521b1-b3dc-450a-9daa-37e51b591d75': '9c187711872d49e481be3cca2277055587d96bf20b982f5550d69b0a567f699b'},
+        "prov:atLocation": "ds011/sub-01/func/sub-01_task-tonecounting_bold_trunctest.nii.gzs"
```

Comment:
This is not related to this PR but there is an extra 's' that I think should be removed, i.e. `sub-01_task-tonecounting_bold_trunctest.nii.gz` instead of `sub-01_task-tonecounting_bold_trunctest.nii.gzs`